std/testing/asserts.ts

```diff
@@ -243,6 +243,64 @@ export function assertStrictEq(
   throw new AssertionError(message);
 }
 
+/**
+ * Make an assertion that `actual` greater than `expected` value. If not
+ * then thrown.
+ */
+export function assertToBeGreaterThan(
+  actual: number,
+  expected: number,
+  msg?: string
+): void {
+  if (actual > expected) {
+    let actualString: string;
+    let expectedString: string;
+    try {
+      actualString = String(actual);
+    } catch (e) {
+      actualString = "[Cannot display]";
+    }
```

Comment:
Use `format()` defined in this module.

Reply:
Hi @nayeemrmn .
Can you please give me an example.

Reply:
I just implement that using format but test failed